src/index.html

```diff
@@ -8,52 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <div class="row stars--0">
-      <div class="row__star"></div>
-      <div class="row__star"></div>
-      <div class="row__star"></div>
-      <div class="row__star"></div>
-      <div class="row__star"></div>
+    <div class="rating stars--0">
+      <div class="rating__star"></div>
+      <div class="rating__star"></div>
+      <div class="rating__star"></div>
+      <div class="rating__star"></div>
+      <div class="rating__star"></div>
     </div>
 
-    <div class="row stars--1">
-      <div class="row__star"></div>
-      <div class="row__star"></div>
-      <div class="row__star"></div>
-      <div class="row__star"></div>
-      <div class="row__star"></div>
+    <div class="rating stars--1">
```

Comment:
you cannot use block modifier without corresponding block

Reply:
i don't understand my mistake about modifier(